src/ride/cable_lift.c

```diff
@@ -475,7 +475,14 @@ int cable_lift_update_track_motion(rct_vehicle *cableLift)
 		rct_vehicle* vehicle = GET_VEHICLE(spriteId);
 		vehicleCount++;
 		frictionTotal += vehicle->friction;
-		var2CTotal += vehicle->acceleration;
+		// Prevent over/underflow in var2CTotal
+		if ((INT32_MAX - abs(vehicle->acceleration)) < abs(var2CTotal) &&
```

Comment:
Cable lift has a fixed acceleration value anyway was there any reason for adding all of this?